spec/inputs/graphite_spec.rb

```diff
@@ -1 +1,54 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/graphite"
+require "logstash/timestamp"
+require "time"
+
+describe LogStash::Inputs::Graphite do
+  before do
+    srand(RSpec.configuration.seed)
+  end
+
+  let(:host) { "127.0.0.1" }
+  let(:port) { rand(5000) + 1025 }
+  let(:queue) { [] }
+
+  let(:client) { TCPSocket.new(host, port) }
+
+  subject { LogStash::Inputs::Graphite.new("host" => host, "port" => port) }
```

Comment:
I understand, but we should otherwise try to standardize the way we start the pipeline, can you wrap this thing you did here in a helper alike to the other? This would be super helpful to others.